about.html

```diff
@@ -0,0 +1,152 @@
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li class="nav-item"><a href="#" class="nav-link">English</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">My Page</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">Logout</a></li>
+          </ul>
+        </div>
+        <div class="header-bottom">
+          <div class="logo"><a href="./index.html">My Logo</a></div>
+          <div class="nav-right">
+            <ul class="main-nav">
+              <li class="nav-item"><a href="#" class="nav-link">About</a></li>
+              <li class="nav-item">
+                <a href="#program" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+              <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+              <li class="nav-item">
+                <a href="#sponsor" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link campain">GC Campain</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <main class="about-wrapper">
+      <section class="about">
+        <div class="about-us">
+          <span class="about-quote"
+            >“Our highest deeds come from helping the lowest people.”</span
+          >
+          <h1 class="about-goal">
+            “Overcoming poverty is not a gesture of charity. It is an act of
+            justice. It is the protection of a fundamental human right, the
+            right to dignity and a decent life.”
+          </h1>
+        </div>
+        <div class="about-ideal">
+          The Genuine Charity brings together the community of experts,
+          academics, nad activists who comprise the Creative Commins affiliates
+          network in a different country every two years. This year, we hope to
+          expand our invitation list including organizations and individuals who
+          want to work with us on shared projects taht advance the cause of yhe
+          commons, free culture nd open kwnoledge. So if you're active and
+          engaged in the worlds of open content and kwnoledge - free software
+          advocates. Wikipedias, Open Kwnoledge, galeries, libraries, museums,
+          archives, governments and foundations, kawyers, and activists - we
+          hope you'll consider joing us this year to build a stronger, more
+          vibrant commons together.
+        </div>
+        <div class="contact">
+          Please contact us per Email for any further questions about Genuine
+          Charity and care at
+          <strong><u>christianbiringanine22@gmail.com</u></strong> >
```

Comment:
- [ ] Good job here but this email is supposed to be centered as shown on the design. It would be great if you could fix that.
- Your design:
![Annotation 2022-04-16 080132](https://user-images.githubusercontent.com/85113567/163665872-c3830413-5d92-47a7-bb11-a51d02f77047.jpg)

- Required design:
![Annotation 2022-04-16 080147](https://user-images.githubusercontent.com/85113567/163665886-3519fd27-b632-4f49-84af-49f39b06de82.jpg)

Reply:
Okay cool!